Modules/Microsoft365DSC/DSCResources/MSFT_AADGroup/MSFT_AADGroup.psm1

```diff
@@ -583,10 +583,6 @@ function Set-TargetResource
                     -Source $MyInvocation.MyCommand.ModuleName
             }
         }
-        if ($assignedLicensesGUIDs.Length -gt 0)
-        {
-            Set-MgGroupLicense -GroupId $currentGroup.Id -AddLicenses $licensesToAdd -RemoveLicenses @()
```

Comment:
Why are we removing the licenses' assignment on the new group?

Reply:
Those lines of code are never executed since $assignedLicensesGUIDs is never initialized. However, licensing is processed in the next block of code that executes when Ensure eq 'Present'